### DISCUSSION TITLE
Microsoft Search  Enable or Not?

### DISCUSSION THREAD

**Discussion by bkndc** [CodyBerenson/PGMA-Modernized#307] [General]
Should Microsoft Search be enabled on Windows Plex Server?  Does Plex use it?   Should certain directories be excluded in the Search configuration?  ie; Plex Database directory?  The Volumes where the movies reside?  Search can be CPU intensive and can cause interrupts.  So, is it needed if the Windows device is only running Plex?  If this has been discussed before I apologize however, I cannot find any mention of it here or on the Plex Forums.

**Comment by JPH71** [comment]
Windows search is not needed for plex.

**Reply by bkndc** [reply]
Thank you.  It seems turning off Microsoft Search or disabling it would be a best practice for "If installing Plex Server on Microsoft Windows".

**Reply by CodyBerenson** [reply]
Agreed.  I just checked my Windows desktop...all AppData is excluded from search...I for sure didn't add it.  

![image](https://github.com/CodyBerenson/PGMA-Modernized/assets/40373801/bbeca165-b474-469a-9ccb-11dd1fd80987)
